transcom/mymove
# B-19982-INT

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19982)

## Summary

Update the services related to PPM document submission to update the appropriate "submitted_" database column for data that is adjustable by closeout counselors, so we can display the data to the customer in the new customer document review component (coming soon!)

[this article](tbd) explains more about the approach used.

### How to test

1. As a customer, submit PPM documents for a trip, pro gear weight, and a storage moving expense.
2. Confirm your data is stored in the related "submitted_" column of the appropriate table
